tests/sqlparser_snowflake.rs

```diff
@@ -2483,6 +2483,37 @@ fn test_sf_trailing_commas() {
     snowflake().verified_only_select_with_canonical("SELECT 1, 2, FROM t", "SELECT 1, 2 FROM t");
 }
 
+#[test]
```

Comment:
Could we move the tests to common and use the [all_dialects_where](https://github.com/apache/datafusion-sqlparser-rs/blob/5f3de315842963e6db47c9f30ddc914dc7c605c9/tests/sqlparser_common.rs#L8254) function? That would make it automatic to add test coverage for other dialects that have this feature enabled in the future